src/qix.js

```diff
@@ -177,14 +140,14 @@ class Qix {
       const globalApi = session.getObjectApi(args);
       globalApi.openApp = globalApi.openDoc =
       (appId, user = '', password = '', serial = '', noData = false) => {
-        session.appId = appId;
-        session.noData = noData;
         config.session.route = '';
         config.appId = appId;
 
         const appSession = Qix.getSession(config);
 
         if (!appSession.apiPromise) {
+          // create a Global API for this session:
+          appSession.getObjectApi(args);
```

Comment:
I noticed that when you do suspend/resume on an `global.openDoc`:d session, `restoreGlobal` would fail since we never generated a global API for that session. When we remove the new session/socket during openDoc, this logic shouldn't be needed.